src/main/java/com/team766/hal/simulator/RobotMain.java

```diff
@@ -62,6 +62,8 @@ public void step() {
 							robot.teleopPeriodic();
 							Scheduler.getInstance().run();
 							break;
+						default:
```

Comment:
If not this, what would an appropriate exception be?
```java
LoggerExceptionUtils.logException(
  new Exception("Unknown robot mode", null);
);
```

Reply:
Probably `IllegalArgumentException` or maybe `UnsupportedOperationException`

The second `null` argument to the exception constructor is not needed, but `ProgramInterface.robotMode` should probably be affixed to the error message so that the invalid value gets recorded in the exception